airflow/executors/local_kubernetes_executor.py

```diff
@@ -48,6 +48,8 @@ class LocalKubernetesExecutor(LoggingMixin):
 
     is_local: bool = False
 
+    serve_logs: bool = True
```

Comment:
You'll need to add a default for the `CeleryKubernetesExecutor` as well (with False being the value I reckon), since that class does not inherit (directly) from BaseExecutor (see [this comment](https://github.com/apache/airflow/issues/28276#issuecomment-1344899475) for more details).

Reply:
i think all Celery* executors need to serve logs, so i'd assume (without having looked) that this one needs to be true (unless something weird happens and  it ends up serving anyway because of its child executor)

Reply:
i think maybe only k8s executor doesn't log serve... we read the pod logs instead (while task running)

Reply:
for `celery executor` the logs are served by the `workers`. 
https://github.com/apache/airflow/blob/3ececb2c79307bd943aad116d7b0b5933af8185a/airflow/cli/commands/celery_command.py#L88-L96

this flag check if the logs need to be served by the `scheduler`.   
should we rename this flag to `scheduler_serve_logs`?

Reply:
> You'll need to add a default for the `CeleryKubernetesExecutor` as well (with False being the value I reckon), since that class does not inherit (directly) from BaseExecutor (see [this comment](https://github.com/apache/airflow/issues/28276#issuecomment-1344899475) for more details).

should we do `hasattr(executor.serve_logs)` to be sure?

Reply:
Oh, durr

Reply:
> > You'll need to add a default for the `CeleryKubernetesExecutor` as well (with False being the value I reckon), since that class does not inherit (directly) from BaseExecutor (see [this comment](https://github.com/apache/airflow/issues/28276#issuecomment-1344899475) for more details).
> 
> should we do `hasattr(executor.serve_logs)` to be sure?


@snjypl 
I'd prefer to have a default added to the CeleryKubernetesExecutor so that code authors in the future don't need to remember/know that access to that flag needs to be protected with hasattr, but if you'd like to do both, then I'm fine with that as well!

Reply:
Done.